qa/tasks/cephfs/test_fscrypt.py

```diff
@@ -21,18 +21,25 @@ def test_fscrypt(self):
         # mounts are never actually tested.
         if isinstance(self.mount_a, FuseMount):
             log.info('client is fuse mounted')
-            self.skipTest('Requires kernel client; xfstests-dev not '\
-                          'compatible with ceph-fuse ATM.')
+            return False
         elif isinstance(self.mount_a, KernelMount):
             log.info('client is kernel mounted')
+            return True
+        log.error('client is unkown mounted')
+        return False
+
+    def test_fscrypt_encrypt(self):
+        if not check_mounter():
+            self.skipTest('Requires kernel client; xfstests-dev not '\
+                          'compatible with ceph-fuse ATM.')
```

Comment:
It'll be simpler and more compact if we can move this call to `skipTest` inside `check_mounted()` . This way we can also avoid code duplication.

In this case the function name, IMO, should be changed to  `skipTest_if_kclient` or `skipTest_if_client_kernel_mounted`.

Reply:
Sound good. I will follow `xfstests-dev` and will rename it to `require_kernel_mount`.